### DISCUSSION TITLE
05/24/2023 Metaverse Standards Forum: Oversight

### DISCUSSION THREAD

**Discussion by funksoup** [omigroup/MSF-Delegates#49] [Oversight]
Metaverse Standards Forum: Oversight
Wednesday, 24 May 2023 from 21:00-22:30 EST (01:00-02:30 UTC) 

**Agenda:** 

- Membership transition update
- Board of Directors update and upcoming elections
- Industrial Metaverse and the Forum (Nokia)
- Outreach update and upcoming opportunities
- Exploratory Group Brainstorming
- Domain Subgroup process discussion
- Domain Group Coordination discussion
- GLTF physics

**--> [Google slides](https://docs.google.com/presentation/d/1Q7VWXU5STfapOIP84xYjVRM-k6kLlGPmLiwtEexCbSM/edit#slide=id.g249c355abd5_2_0)<--**

**--> [Slides as PDF](https://github.com/omigroup/MSF-Delegates/files/11560342/Metaverse.Standards.Forum.Oversight.17.24May23.pdf)<--**

![Slide4](https://github.com/omigroup/MSF-Delegates/assets/34685781/d9793c3f-6d9a-47c6-87fc-374a624570af)

**Comment by funksoup** [comment]
Membership transition period has started. 
- Six-month transition period for all members to execute updated membership agreement
- All Members are encouraged to execute the new online agreements ASAP:
--  Very similar legal obligations to initial agreements
-- https://metaverse-standards.org/members/  

Deadlines to note:
- Sign the new membership agreement by **October 2023**
- HOWEVER, if we want to nominate someone from our community to the MSF board, then we need to sign the new membership agreement asap and complete the nomination form by **June 24, 2023** (see slide below on board election)


![Slide5](https://github.com/omigroup/MSF-Delegates/assets/34685781/075511cf-185f-4e9a-be5b-4310dfa2c383)


![Slide6](https://github.com/omigroup/MSF-Delegates/assets/34685781/0fa68b17-d3ca-4283-8892-d5fa3a5400ae)


![Slide7](https://github.com/omigroup/MSF-Delegates/assets/34685781/91dec71f-58ec-4a9c-8658-ad4c0272087a)


![Slide8](https://github.com/omigroup/MSF-Delegates/assets/34685781/8827c852-69a3-4e83-bfac-f9dc7b81a8ae)


**Board nomination info:**
![Slide9](https://github.com/omigroup/MSF-Delegates/assets/34685781/2d0c22be-8ae6-402e-b5fe-36bccd5892a7)

**Comment by funksoup** [comment]
![Slide10](https://github.com/omigroup/MSF-Delegates/assets/34685781/6e182d88-991a-4baf-b54f-d1348b809183)


Nokia gave a presentation that focused on the *industrial* metaverse:
- [Slide deck](https://github.com/omigroup/MSF-Delegates/files/11560388/Nokia.presentation.to.MSF.oversight.on.industrial.metaverse.May.2023.FINAL.pdf)
- [Recording of Nokia's presentation](https://tools.metaverse-standards.org/documents/f/9404)

(No decisions or actions steps needed; just something to be aware of i.e., growth of industrial focus and how MSF can be involved)

The detailed roadmap is very interesting (slide 7):

![roadmap_Nokia_MetaverseDeployment](https://github.com/omigroup/MSF-Delegates/assets/34685781/b6414e5c-e5b4-47be-b01f-70300daf4eed)

**Comment by funksoup** [comment]
![Slide11](https://github.com/omigroup/MSF-Delegates/assets/34685781/34f606fd-4cbe-435e-917a-1e3cd2ebd379)


![Slide12](https://github.com/omigroup/MSF-Delegates/assets/34685781/b2043fa6-fa80-4e53-a997-53f844d5f5b3)


![Slide13](https://github.com/omigroup/MSF-Delegates/assets/34685781/5a204b5d-5005-4b6d-b71c-9c257fb8377e)


![Slide14](https://github.com/omigroup/MSF-Delegates/assets/34685781/52224aec-57dc-4860-99c1-5c65b154367d)


![Slide15](https://github.com/omigroup/MSF-Delegates/assets/34685781/b0a2b904-fbd6-45ab-9ed6-b52f325d7859)

**Reply by JoyceBettencourt** [reply]
After the June 14th OMI meeting, several of us (@funksoup @aaronfranke @indiebio and myself) discussed the impending deadline to refile OMI's application to MSF now they are incorporated and going through the process of getting principal dues waived.

**Comment by funksoup** [comment]
I (@funksoup) asked to add an item to the agenda concerning GLTF physics (via @aaronfranke) as described in this [GLTF Physics Standard page](https://github.com/omigroup/omigroup/discussions/393). (Special thanks to [IndieBio](https://twitter.com/indiebio) for the table creation).

We displayed the table and talked through differences between the two competing standards, highlighting details such as the differences between body category and materials definition. We got a great response from the MSF audience at the meeting, and we were encouraged to ping the larger email list to get more feedback.

**(Short link to the page, for easier sharing: https://bit.ly/gltfstandard)**
